spec/system/admin/admin_bans_or_warns_user_spec.rb

```diff
@@ -26,7 +26,7 @@ def warn_user
   end
 
   def add_tag_moderator_role
-    tag = FactoryBot.create(:tag)
```

Comment:
We include `FactoryBot::Syntax::Methods` in our `rails_helper.rb` so we don't need to call through the module.